onnx/defs/nn/defs.cc

```diff
@@ -202,23 +204,23 @@ computes the output.)DOC";
                 "Constrain input and output types to float tensors.");
             schema.Attr("kernel_shape",
                         "The shape of the convolution kernel.",
-                         AttributeProto::INTS, OPTIONAL);
+                         AttributeProto::INTS);
```

Comment:
this is optional in this case, right? it's even not needed. the kernel shape may be inferred from input "W".

Reply:
Oh, you are right, I didn't think to this